src/components/Quotes.js

```diff
@@ -0,0 +1,52 @@
+import './Quotes.css';
+import { useEffect, useState } from 'react';
+
+function Quotes() {
+  const [quote, setQuote] = useState({});
+
+  const API_KEY = 'zWbFdXe3BQWrlhFsnImSAw==HEDkeSdVW05kaUH7';
+  useEffect(() => {
+    fetch('https://api.api-ninjas.com/v1/quotes?category=knowledge', {
+      headers: { 'X-Api-Key': API_KEY },
+      method: 'GET',
+      contentType: 'application/json',
+    }).then((response) => response.json()).then((result) => {
+      if (result.error) {
+        setQuote({ quote: 'An Error Occurred' });
+      } else {
+        setQuote(result[0]);
+      }
+    });
+  }, []);
+
```

Comment:
- [ ] Good job so far 👏 👏 As it is in the project requirements, you are expected to have 2 more states on top of the one you have here. These are `loading` state and `error` state. You can see it 👇 
![image](https://user-images.githubusercontent.com/90222110/227539975-6af8c417-b2bf-4342-8d00-2cd5d6681f8c.png)

These states will help you know how to handle the response from the request in terms of the View or Interface.  As you understand, this will require you to track different case of the response from the API. Below I will provide pieces of code that can help you to implement the suggested changes well. 😃 so please refer to it while doing implementation 👇 

```js
function Quotes() {
  const [quotes, setQuotes] = useState([])
  const [hasError, setHasError] = useState(false)
  const [isLoading, setIsLoading] = useState(false)

  useEffect(() => {
    const fetchQuotes = async () => {
      setIsLoading(true)
      try {
        const res = await fetch(
          "https://api.api-ninjas.com/v1/quotes?category=happiness",
          {
            method: "GET",
            headers: {
              "X-API-Key": "API_KEY",
            },
          }
        )
        const json = await res.json()
        setQuotes(json)
      } catch (error) {
        setHasError(true)
      }
      setIsLoading(false)
    }
    fetchQuotes()
  }, [setQuotes, setIsLoading])

  if (hasError || Number(quotes.length) === 0)
    return (
// RETURN WHAT SHOULD BE SEEN WHEN THERE IS ERROR
)

  if (isLoading) return (
// RETURN THE OTHER SPINNERS YOU HAVE
)

  return (
   // RETURN YOUR QUOTE HERE
  )
}

```